packages/modal/src/ModalHeader.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Flex, CloseButton, Text } from 'pcln-design-system'
+
+const HeaderWrapper = styled(Flex)`
+  height: 40px;
+`
+
+const StyledCloseButton = styled(CloseButton)`
+  margin-right: 8px;
```

Comment:
would an inline mr={2} work instead?

Reply:
I thought the button doesn't accept mr, will update